bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TupleValueImpl.java

```diff
@@ -827,4 +829,25 @@ private void resetSize(int index) {
             size = index + 1;
         }
     }
+
+    private void validateTupleSizeAndInherentType() {
+        int numOfMandatoryTypes = this.tupleType.getTupleTypes().size();
+        // Check if tuple has at least one more members than the number of mandatory type to compensate the
+        // removed member
```

Comment:
We can remove this comment; it is unnecessary.

Reply:
Resolved with 717906f159d143fc9b5c9c2fb35042521b60f2a0